networking/eltnam/setup_routers.py

```diff
@@ -0,0 +1,33 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+import sys
+import time
+
+teamNum = sys.argv[1]
+
+wd = webdriver.Firefox()
```

Comment:
what happens if no firefox? or no internet access? whole script errors? 

If this is part of a larger devops script, how are they handling that? Are they aware of possible outputs, failure states this might generate? 

It's gonna be annoying if they spend 3 hours flashing and running scripts on everything, then realize they have to do it again cause a subscript somewhere broke the whole thing.